microsoftgraph/microsoft-graph-explorer-v4
# Enhancement: Add copy button confirmation

## Overview

Fixes #1103 
This enhances the UI/UX by ensuring a user receives a confirmation when they click the copy button at either of these areas
- The Access Token tab
- The Code snippets tab
- The Adaptive Cards (JSON template) tab
- The Share button on the query response area

### Demo
before clicking copy icon
![image](https://user-images.githubusercontent.com/18407044/144446481-77b46f7f-9be9-4f35-864e-59d3634eec4a.png)

after clicking copy Icon
![image](https://user-images.githubusercontent.com/18407044/144446618-97911ece-96ae-4493-8a4c-b4ffeac2d3a2.png)


### Notes
These icons have a set timer and reset after 3 seconds when clicked.
Tooltip changes from 'Copy' to 'Copied'

## Testing Instructions

* Load Graph Explorer
* Navigate to the tabs mentioned above
* Click on the copy button and observe the enhancement.
